packages/core/src/rules/use_bool_expression.ts

```diff
@@ -0,0 +1,95 @@
+import {Issue} from "../issue";
+import * as Statements from "../abap/2_statements/statements";
+import * as Expressions from "../abap/2_statements/expressions";
+import * as Structures from "../abap/3_structures/structures";
+import {ABAPRule} from "./_abap_rule";
+import {ABAPFile} from "../files";
+import {BasicRuleConfig} from "./_basic_rule_config";
+import {Version} from "../version";
+import {IRuleMetadata, RuleTag} from "./_irule";
+
+// note this rule assumes abap_true and abap_false is used for boolean variables
+// some other rule will in the future find assignments to abap_bool that are not abap_true/abap_false/abap_undefined
+
+export class UseBoolExpressionConf extends BasicRuleConfig {
+}
+
+export class UseBoolExpression extends ABAPRule {
+  private conf = new UseBoolExpressionConf();
+
+  public getMetadata(): IRuleMetadata {
+    return {
+      key: "use_bool_expression",
+      title: "Use boolean expression",
+      shortDescription: `Use boolean expression, xsdboolc from 740sp08 and up, boolc from 702 and up`,
```

Comment:
typo here, xsdboolc

Reply:
thanks

Reply:
fixed in https://github.com/abaplint/abaplint/pull/1345/commits/ec75bab8a61b77d8787acf6a86fa9d840b623145